ffiler

```diff
@@ -102,53 +99,53 @@ shift $((OPTIND-1))
 # bomb unless the user has told us how to sort the file(s)
 [[ -z "$fileby" ]] && { bomb "Undefined sort method"; }
 
-for X in $@ ; do
+for X in "$@"; do
   fname="${1:-}"
   shift
 
   # validate the given user input
-  [[ -z "${fname}" ]]   && { usage; exit -1; }
```

Comment:
Really? What version of bash are you running?

    mattcen@sonar:tmp$ fname="foo bar"; [[ -z $fname ]] || echo SUCCESS
    SUCCESS           
    mattcen@sonar:tmp$